front/src/modules/simulationResult/components/SpaceTimeChartWrapper/useLazyProjectTrains.ts

```diff
@@ -44,28 +40,32 @@ const useLazyProjectTrains = ({
     );
   }, []);
 
+  const isProjectionSuccessful = isProjectionWithPathfinding(projectionPath);
+
   useEffect(() => {
-    if (!path) return undefined;
     const opRefs: OperationalPointReference[] = [];
     const opDistances: number[] = [];
+
+    const opsFromProjection = projectionPath.operationalPoints;
```

Comment:
If you do this change, you can remove the ? on the lines below
```suggestion
    const { operationalPoints = [] } = projectionPath;
```

Reply:
Done